public/i18n/qqq.json

```diff
@@ -7,6 +7,8 @@
 	},
 	"query-builder-heading": "Title of the page",
 	"query-builder-intro-text": "Intro text describing what query builder is.",
+	"query-builder-query": "Name of the section for building the query",
```

Comment:
"Name" is a bit unspecific with respect to how it is used. How about "Heading"? Maybe we could also include that in the message-key?

Reply:
Yeah, good point. Added to both message documentation and message name